src/ansible_compat/runtime.py

```diff
@@ -116,14 +117,19 @@ def clean(self) -> None:
             shutil.rmtree(self.cache_dir, ignore_errors=True)
 
     def exec(
-        self, args: Union[str, List[str]], retry: bool = False
+        self, args: Union[str, List[str]], retry: bool = False, tee: bool = False
     ) -> CompletedProcess:
         """Execute a command inside an Ansible environment.
 
         :param retry: Retry network operations on failures.
         """
+        if tee:
+            runner: Callable[..., CompletedProcess] = subprocess_tee.run
```

Comment:
Don't call it runner. By invoking it, it doesn't *just* create an object but also invokes commands. It must be a verb.

Reply:
`run_func` ?